ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/AudioPlayer.qml

```diff
@@ -1,107 +1,82 @@
 import QtQuick 2.3
-import QtMultimedia 5.14
-import "../../../../../shared"
+import QtQuick.Layouts 1.13
+import QtQuick.Controls 2.13
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import "../../../../Media"
 import "../../../../../imports"
 
-Item {
+
+Rectangle {
+    id: root
```

Comment:
Why its called `root`? its not a root object.

Reply:
https://github.com/Furkanzmc/QML-Coding-Guide#cs-5-giving-components-ids

> Make sure that the top most component in the file always has root as its id.

Reply:
Sorry, but Who is Furkanzmc?

Reply:
It's official Qt Coding Guide?

Reply:
We should clarify this with @alexandraB99 probably.

Reply:
BTW, try searching all Qt examples (qml) for `id: root`.